near-bindgen/src/environment/env.rs

```diff
@@ -79,10 +79,7 @@ pub fn read_register(register_id: u64) -> Option<Vec<u8>> {
 /// Returns the size of the register. If register is not used returns `None`.
```

Comment:
That's a lot of reformating. We should figure out why our rustfmt format things differently. Do you have https://github.com/nearprotocol/nearcore/blob/master/rustfmt.toml in ~/.config/rustfmt/ ?

Reply:
Yes. I think it's because the original file was not reformatted at all. Rustfmt is enabled per project in clion

Reply:
My rustfmt was uninstalled :(